### DISCUSSION TITLE
Database error on new install

### DISCUSSION THREAD

**Discussion by iammattmartin** [outline/outline#6675] [Self Hosting]
Trying to install as per the docs and after running `docker compose run --rm outline yarn db:create --env=production-ssl-disabled` this is the error.

```
$ sequelize db:create --env=production-ssl-disabled

Sequelize CLI [Node: 20.11.1, CLI: 6.6.1, ORM: 6.35.2]

Loaded configuration file "server/config/database.json".
Using environment "production-ssl-disabled".

ERROR: connect ECONNREFUSED 127.0.0.1:5432

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

The db string in the env and docker file are as documented and match. SQL is up and answering on the port.

Any help would be appreciated.

**Comment by iammattmartin** [comment]
Found the answer with better searches. Seems this has been hit before https://github.com/outline/outline/discussions/5411 and looks like the install docs may need a review. :)

**Reply by tommoor** [reply]
Is it also answered here?
https://docs.getoutline.com/doc/troubleshooting-HXckrzCqDJ#h-cannot-access-postgres-redis
